cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -1,10 +1,13 @@
 DROP TABLE IF EXISTS tablea cascade;
 DROP TABLE IF EXISTS tableb cascade;
+DROP TABLE IF EXISTS tablec cascade;
+DROP TABLE IF EXISTS tabled cascade;
 CREATE EXTENSION IF NOT EXISTS pg_trgm;
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 CREATE TABLE tablea (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
 CREATE TABLE tableb (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
-
+CREATE TABLE tablec (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
+CREATE TABLE tabled (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
```

Comment:
Can you remove the lines with the extension unaccent and the function f_unaccent? The compound index tests do not use it.